0.download_data/scripts/nbconverted/download_data.py

```diff
@@ -10,38 +10,57 @@
 import pandas as pd
 
 
-# ### Specify version of mitocheck_data to download from
+# ## Specify mitocheck_data types and paths
+# 
+# We use multiple types of data for model training/testing.
 
 # In[2]:
 
 
-labeled_data_hash = "20369033f579dca1334cb2c58a1c6d532322f93e"
-labeled_data_url = f"https://raw.github.com/WayScience/mitocheck_data/{labeled_data_hash}/3.normalize_data/normalized_data/training_data.csv.gz"
+labeled_data_hash__ic = "20369033f579dca1334cb2c58a1c6d532322f93e"
+labeled_data_url__ic = f"https://raw.github.com/WayScience/mitocheck_data/{labeled_data_hash__ic}/3.normalize_data/normalized_data/training_data.csv.gz"
 
+# ADD PATH TO DATA FROM GITHUB
+# labeled_data_hash = ""
```

Comment:
consider removing if not in use

Reply:
I will leave so we know where to clearly add this in the future.